content/articles/ssh-tunneling/index.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /ssh-tunneling/
+title: SSH Tunneling
+description: This article will discuss the concept of SSH tunneling. SSH tunneling is a technique that allows us to access remote resources that we do not have access to because they are internal to that network.
+author: joseph-ongoma
+date: 2022-02-22T00:00:00-12:20
+topics: [Networking]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/ssh-tunneling/hero.jpg
+    alt: SSH Tunneling Hero Image
+---
+Secure Shell (SSH) tunneling, also known as SSH port forwarding, is a technique for sending arbitrary data over an encoded SSH connection. SSH tunnels enable connections to a local port (i.e., a port on your desktop) to be transferred to a remote computer across a secure channel. 
+<!--more-->
+SSH tunneling allows us to access remote resources that we do not have access to because they are internal to that network. SSH tunneling is also used to allow others outside our network to have access to it. It is done through TCP tunneling.
+
+To perform SSH tunneling, we will need to perform port forwarding to allow port 22. Port 22 is the SSH port on our router to be routed to our public SSH server.
+
+To ensure the security of our network services, not all ports are directly accessible from outside the Enterprise Network Compute System (ENCS) network. If you are offsite and need to access a protected resource, you can use SSH to tunnel through an accessible resource to reach the protected resource.
+
+In this article, we will look at two main subtopics in SSH tunneling, that is:
+- Remote port forwarding
+- Local port forwarding
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [Local port forwarding](#local-port-forwarding)
+- [Remote port forwarding](#remote-port-forwarding)
+- [Conclusion](#conclusion)
+
+### Prerequisites
```

Comment:
Isn't an understanding of networking required as a prerequisite? 
Add a link to any networking article/published content that is available which the reader should look at before proceeding with this article.